create/index.html

```diff
@@ -17,7 +17,6 @@
     <header>
```

Comment:
Really cool decision to take out the create link when you're on the create page! Makes total sense and is one of those little details that shows holistic thinking. Well done!